packages/node_modules/@ciscospark/widget-recents/README.md

```diff
@@ -0,0 +1,274 @@
+# Spark Recents Widget _(@ciscospark/widget-recents)_
+
+* THIS WIDGET CONTAINS EXPERIMENTAL CODE *
+
+> The Spark Recents widget allows developers to easily incorporate a list of recent Cisco Spark calls, 1 to 1 messages and spaces into an application.
+
+
+## Table of Contents
+-   [Background](#background)
+-   [Install](#install)
+    -   [CDN](#cdn)
+    -   [Build from Source](#build-from-source)
+-   [Usage](#usage)
+    -   [Quick Start](#quick-start)
+    -   [HTML](#html)
+      -   [Data API](#data-api)
+      -   [Browser Globals](#browser-globals)
+    -   [JSX](#jsx)
+    -   [Events](#events)
+-   [Browser Support](#browser-support)
+
+
+## Background
+
+This widget handles the heavy lifting of coordinating between your application and the Spark APIs, and provides components of the Spark recents lists experience without having to build all front end UI yourself.
+
+Our widget is built using React <https://github.com/facebook/react>, Redux <https://github.com/reactjs/redux>, and the Spark Javascript SDK <https://github.com/ciscospark/spark-js-sdk>.
```

Comment:
"Cisco Spark JavaScript SDK"

Any reason you're showing the urls instead of using markdown links?